.github/workflows/qa.yml

```diff
@@ -88,7 +88,14 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        php: [7.2, 7.3, 7.4, 8.0]
+        php: [7.1, 8.0]
```

Comment:
I think you can revert the matrix, I will restart the failed builds manually

Reply:
Done